### PR TITLE
Updated serverless delivery setup guide

### PR DESCRIPTION
**Pull request by avasquez614**
Ticket craftercms/craftercms#3716

### CODE REVIEW

File: source/system-administrators/activities/delivery/setup-serverless-delivery.rst [hunk 1]
```diff
@@ -0,0 +1,390 @@
...
+
+----------------------------------------------------
+Step 4: Create the Site in the Authoring Environment
+----------------------------------------------------
+
+#. Login to the Crafter Studio in the authoring environment from your browser.
+#. Click the ``Create Site`` button
+#. Choose the ``Editorial`` blueprint, enter the ``Site Id`` (e.g. ``editorial``), and then review and create.
+#. Go to your AWS console in your browser and on the ``Services`` dropdown search for CloudFormation. You should then 
+   see  the CloudFormation for the site you just created with the status ``CREATE_IN_PROGRESS``. After a several
```
Comment (alhambrav) [inline source/system-administrators/activities/delivery/setup-serverless-delivery.rst]: Typo? Remove `a` so the sentence goes like: `After several minutes, the ...`

File: source/system-administrators/activities/delivery/setup-serverless-delivery.rst [hunk 2]
```diff
@@ -0,0 +1,390 @@
...
+   minutes, the status should change to ``CREATE_COMPLETE``, which tells the Crafter Deployer that the it is able to
```
Comment (alhambrav) [inline source/system-administrators/activities/delivery/setup-serverless-delivery.rst]: Typo? Remove `the` so it goes like: `... which tells the Crafter Deployer that it is able to ...`

File: source/system-administrators/activities/delivery/setup-serverless-delivery.rst [hunk 3]
```diff
@@ -0,0 +1,390 @@
...
+
+- An AWS account
+- A Crafter CMS authoring environment
+- A Crafter CMS delivery environment
+
+--------------------------------------------------------------
+Step 1: Create an Elasticsearch Domain for Delivery (optional)
+--------------------------------------------------------------
+
+Since serverless delivery requires a single Elasticsearch endpoint readable by all Engine instances, we recommed you
```
[the remark this answers is not in the document]
Comment (avasquez614) [reply source/system-administrators/activities/delivery/setup-serverless-delivery.rst]: Done

File: source/system-administrators/activities/delivery/setup-serverless-delivery.rst [hunk 4]
```diff
@@ -0,0 +1,390 @@
...
+.. important:: Authoring can also use an Elasticsearch domain, but be aware that in a clustered authoring environment
+               each authoring instance requires a separate Elasticsearch instance. If you try to use the same ES domain
+               then you will have multiple preview deployers writing to the same index.
+
+To create an AWS Elasticsearch domain please do the following:
+
+#. In the top navigation bar of your AWS console, click the ``Services`` dropdown menu, and search for 
+   ``Elasticsearch Service``.
+#. Click on ``Create a new domain``.
+#. Select a ``Deployment Type`` and on the Elasticsearch version, pick ``6.7``.
```
[the remark this answers is not in the document]
Comment (avasquez614) [reply source/system-administrators/activities/delivery/setup-serverless-delivery.rst]: Done

File: source/system-administrators/activities/delivery/setup-serverless-delivery.rst [hunk 5]
```diff
@@ -0,0 +1,390 @@
...
+
+#. Make sure that the you have an application load balancer (ALB) fronting the Delivery Engine instances and that it's 
+   accessible by AWS CloudFront.
+
+-----------------------------------------------------
+Step 3: Configure Authoring for Serverless Deployment
+-----------------------------------------------------
+
+Instead of having one Crafter Deployer per node in delivery, for serverless you just need a single Deployer uploading
+files to S3. The authoring preview deployer can be used also for serverless deployment, when there's only one
```
[the remark this answers is not in the document]
Comment (avasquez614) [reply source/system-administrators/activities/delivery/setup-serverless-delivery.rst]: Done

File: source/system-administrators/activities/delivery/setup-serverless-delivery.rst [hunk 6]
```diff
@@ -0,0 +1,390 @@
...
+    
+    studio.serverless.delivery.deployer.target.template.params:
+      # The delivery Elasticsearch endpoint (optional is authoring is using the same one, specified in the ES_URL env variable)
+      elastic_search_url: https://vpc-serverless-test-jpwyav2k43bb4xebdrzldjncbq.us-east-1.es.amazonaws.com    
+
+- When using the ``aws-cloudformed-s3`` target template (the default one), the Deployer creates first an AWS 
+  CloudFormation stack with an S3 bucket where the site content will be uploaded and a CloudFront that will serve 
+  ``/static-assets`` directly and will redirect any other requests to the Delivery Engine LB (which you specify in 
+  ``studio.serverless.delivery.deployer.target.template.params.aws.cloudformation.deliveryLBDomainName``).
+- The ``aws.cloudformation.namespace`` is basically the prefix of the S3 bucket mentioned in hte previous step. This
```
[the remark this answers is not in the document]
Comment (avasquez614) [reply source/system-administrators/activities/delivery/setup-serverless-delivery.rst]: Done

File: source/system-administrators/activities/delivery/setup-serverless-delivery.rst [hunk 7]
```diff
@@ -0,0 +1,390 @@
...
+           ...
+           cloudformation:
+             # AWS access key (optional if aws.accessKey is specified)
+             access_key: XXXXXXXXXX
+             # AWS secret key (optional if aws.secretKey is specified)
+             secret_key: XXXXXXXXXXXXXXXXXXXX
+
+- By default, the CloudFront created by Deployer will have a ``*.cloudfront.net`` domain name. To have CloudFront use 
+  additional domain name(s) please specify the AWS ARN of the domain SSL certificate (``cloudfrontCertificateArn``) and 
+  the altername domain name(s) (``alternateCloudFrontDomainNames``):
```
[the remark this answers is not in the document]
Comment (avasquez614) [reply source/system-administrators/activities/delivery/setup-serverless-delivery.rst]: Done

File: source/system-administrators/activities/delivery/setup-serverless-delivery.rst (hunk 1 above)
Comment (avasquez614) [reply source/system-administrators/activities/delivery/setup-serverless-delivery.rst]: Done

File: source/system-administrators/activities/delivery/setup-serverless-delivery.rst (hunk 2 above)
Comment (avasquez614) [reply source/system-administrators/activities/delivery/setup-serverless-delivery.rst]: Done

**Pull request merged**